blocks/color-palette/index.js

```diff
@@ -61,7 +61,7 @@ export function ColorPalette( { colors, disableCustomColors = false, value, onCh
 							color={ value }
 							onChangeComplete={ ( color ) => onChange( color.hex ) }
 							style={ { width: '100%' } }
-							disableAlpha
```

Comment:
Thanks for your work on this PR but I have a concern here.

If we were to introduce this prop, why should we use it only in the `ChromePicker` and not for the palette colors as well?

Also, This doesn't affect the color value, since we only pass the `color.hex` value on the `onChange` call.

---
Can't we just keep this component as is and use a separate `RangeControl` if we want to tweak the alpha value? Alternatively, you can still use the `ChromePicker` directly in your components?

Reply:
I feel like both of the points about extending `ChromePicker` by yourself and `RangeControl` are valid. 

My only point towards adding is more on the side of just making the internal Components more re-usable and less of _copy_ the whole file into your project because some params are not exposed.

Will apply  `onChange` modification, if you think it's worth making internal components more extendable _just because_.